e2e/utils.go

```diff
@@ -1688,6 +1685,25 @@ func k8sVersionGreaterEquals(c kubernetes.Interface, major, minor int) bool {
 	return (vMajor > major) || (vMajor == major && vMinor >= minor)
 }
 
+var brokenMetrics *bool = nil
+
+// k8sBrokenMetrics detects if Kubernetes 1.34 is used, this version has broken
+// kube_volume_stats_* metrics:
+// https://github.com/kubernetes/kubernetes/issues/133847
+func k8sBrokenMetrics(c kubernetes.Interface) bool {
+	if brokenMetrics == nil {
+		broken := true
+		functional := true
+		if k8sVersionGreaterEquals(c, 1, 34) && !k8sVersionGreaterEquals(c, 1, 35) {
```

Comment:
lets open a tracker to revert it back even if its fixed in some patch 1.34 release

Reply:
Done with #5635.